ibis-server/tests/routers/v2/connector/test_postgres.py

```diff
@@ -148,6 +148,23 @@ def test_query_with_connection_url(manifest_str, postgres: PostgresContainer):
         assert result["data"][0][0] == 1
         assert result["dtypes"] is not None
 
+    def test_query_with_dot_all(manifest_str, postgres: PostgresContainer):
+        connection_info = _to_connection_info(postgres)
+        response = client.post(
+            url=f"{base_url}/query",
+            params={"limit": 1},
+            json={
+                "connectionInfo": connection_info,
+                "manifestStr": manifest_str,
+                "sql": 'SELECT "o".* FROM "Orders" AS "o"',
+            },
+        )
```

Comment:
Could you add the end-to-end SQL test for the calculated field and join syntax? Maybe we should add tested in `TestDynamicFields`, too. We need to make sure the model-generating behavior is fine in `WrenSqlRewrite`.